kukur/base.py

```diff
@@ -24,13 +24,30 @@ class Dictionary:
 
 
 @dataclass
-class ComplexSeriesSelector:
+class SeriesSelector:
```

Comment:
For backwards compatibility in APIs, I think we should add an `@property` getter for `.name`. That could replace the `get_series_name` method, because the implementation will be the same.